### PR TITLE
DropButton stories changed to new structure

### PR DESCRIPTION
**Pull request by leossantiago**
#### What does this PR do?

Update DropButton stories to Controls/DropButton.

Note: the new structure creates a conflict with `const Calendar` and `Calendar` component, so I renamed this story when exporting, but named it back to calendar in `DropButton.stories.js` using `export { CalendarDrop as Calendar }`. 

#### Where should the reviewer start?

`components/DropButton/stories/DropButton.stories.js`

#### What testing has been done on this PR?

Storybook

#### How should this be manually tested?

Storybook

#### What are the relevant issues?

#4651

#### Screenshots (if appropriate)

image

#### Do the grommet docs need to be updated?

No.

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible.

### CODE REVIEW

**Review by ShimiSun — APPROVED** [review]

**Comment by ShimiSun** [comment]
@leossantiago hi buddy, thanks for this PR! I noticed that Chromatic is still picking up the stories of this PR, while all DropButton stories should be excluded (you can add the Calendar story to be excluded as well). Can you please take a look at the issue? might be something with  the syntax.

**Comment by leossantiago** [comment]
Hi Shimi @ShimiSun ! I'll work on that asap. Thanks for the note! Out of curiosity, how should one know if a story should have this chromatic disable parameter? And also, how should I test if the stories are being excluded properly? 

I've been learning a lot from Grommet's codebase, and I'd love to understand this further. Also, it would help me on, maybe, future contributions aswell :smile:

**Comment by ShimiSun** [comment]
Hi buddy,
>Out of curiosity, how should one know if a story should have this chromatic disable parameter? 

Since the previous code had it, it should pass along to the new code structure, we shouldn't lose it, it will save us time on the deployment of the chromatic tools, and save us screenshots ($$$)

> And also, how should I test if the stories are being excluded properly?

Make sure you are following the right syntax as presented in other examples.

>I've been learning a lot from Grommet's codebase, and I'd love to understand this further. Also, it would help me on, maybe, future contributions aswell 😄

Let me know if you'd like to have a quick call so I can give a little more background.

File: src/js/components/DropButton/stories/Menu.js [hunk 1]
```diff
@@ -44,6 +43,7 @@ const MenuDropButton = () => {
   );
 };
 
-storiesOf('DropButton', module).add('Menu', () => <MenuDropButton />, {
+export const Menu = () => <MenuDropButton />;
+Menu.parameters = {
```
Comment (shabtaisharon) [inline src/js/components/DropButton/stories/Menu.js:47]: Hi @leossantiago, Here is the correct syntax to disable chromatic on a story:
```
Menu.story = {
  parameters: {
    chromatic: { disable: true },
  },
};
```

File: src/js/components/DropButton/stories/Simple.js [hunk 2]
```diff
@@ -47,6 +46,7 @@ const SimpleDropButton = () => {
   );
 };
 
-storiesOf('DropButton', module).add('Simple', () => <SimpleDropButton />, {
+export const Simple = () => <SimpleDropButton />;
+Simple.parameters = {
```
Comment (shabtaisharon) [inline src/js/components/DropButton/stories/Simple.js:50]: Same here
```
Simple.story = {
  parameters: {
    chromatic: { disable: true },
  },
};
```

File: src/js/components/DropButton/stories/Menu.js (hunk 1 above)
Comment (leossantiago) [reply src/js/components/DropButton/stories/Menu.js]: Thanks! I'll fix that :heart:

**Comment by leossantiago** [comment]
@ShimiSun I think it's all good now. Also, thanks to @shabtaisharon for the fix suggestion. Much appreciated :heart: 
Let me know if there's anything else that still needs a fix on this PR!

**Review by ShimiSun — APPROVED** [review]
Obrigado! ❤️

**Pull request merged**